contrib/pylightning/lightning/lightning.py

```diff
@@ -329,31 +329,29 @@ def _deprecated_close(self, peer_id, force=None, timeout=None):
     def close(self, peer_id, *args, **kwargs):
         """
         Close the channel with peer {id}, forcing a unilateral
-        close after {unilateraltimeout} seconds if non-zero.
+        close after {unilateraltimeout} seconds if non-zero, and
+        the to-local output will be sent to {destination}.
 
         Deprecated usage has {force} and {timeout} args.
         """
-        unilateraltimeout = None
 
         if 'force' in kwargs or 'timeout' in kwargs:
             return self._deprecated_close(peer_id, *args, **kwargs)
 
         # Single arg is ambigious.
```

Comment:
This does not check a single arg anymore

Reply:
Good catch!
Here should also adds:
```
            if len(args) == 2:
                if args[0] == None and isinstance(args[1], int):
                    return self._deprecated_close(peer_id, *args, **kwargs)
```